README.md

```diff
@@ -160,8 +160,20 @@ Run `make build-all`.
 ## Try Out the Application
 
 Once your application is deployed, you can try it out by navigating to `https://$DOMAIN`,
-where `$DOMAIN` is the custom domain
-you configured in `env.mk`.
+where `$DOMAIN` is the custom domain you configured in `env.mk`.
+
+### Setup first admin user
+
+After you login at least once to the app, you can use this script to make your
+account an `admin`. Afterwards you'll be able to use the Users page to manage
+other accounts. To use this script you will need to follow the guide of
```

Comment:
minor nit: "initialize" is a verb so I suggest changing this to at least
```suggestion
other accounts. To use this script you will need to follow the guide to
```
or more simply:
```suggestion
other accounts. To use this script you will need to
```